Dockerfile

```diff
@@ -0,0 +1,34 @@
+#ARG INSTALL_PYTHON_VERSION=${INSTALL_PYTHON_VERSION:-3.8.12}
+#FROM python:${INSTALL_PYTHON_VERSION}-slim-bullseye AS base
+# FROM debian:bullseye-slim AS base
+FROM ssthapa/django5:0.1
+
+MAINTAINER Suchandra Thapa <suchandra.spam+docker@gmail.com>
+
+ENV PYTHONFAULTHANDLER=1 \
+    PYTHONUNBUFFERED=1 \
+    PYTHONHASHSEED=random \
+    PIP_NO_CACHE_DIR=off \
+    PIP_DISABLE_PIP_VERSION_CHECK=on \
+    PIP_DEFAULT_TIMEOUT=100
+
+
+COPY pyproject.toml /srv/django/pyproject.toml
+
+RUN apk update && \
+    apk add rust cargo qpdf qpdf-dev poppler poppler-dev g++ gdal geos alpine-sdk \
+            imagemagick imagemagick-dev pango nodejs npm yarn
+
+ENV PYTHONPATH "${PYTHONPATH}:/srv/django"
+WORKDIR /srv/django
+COPY ./froide /srv/django
+RUN cd froide && yarn link && yarn link froide && yarn install
+RUN /root/.local/bin/poetry install
+
+
+COPY local_settings.py /srv/django/froide/local_settings.py
```

Comment:
We won't need to do this anymore - a Dockerfile matching the steps of https://github.com/codewithaloha/uipa?tab=readme-ov-file#instructions-1 should do the trick (with something handling the edge case of the first initial seed of the data)

Reply:
The settings file doesn't seem to pickup the elasticsearch host from env variables.  So I need to have a copy of the settings that tells django to try to connect to es on the network instead of localhost. I need to debug this a bit more but I figured this works for now so people could use this rather than waiting longer.